src/librepgp/stream-packet.cpp

```diff
@@ -1374,169 +1378,172 @@ signature_parse_subpacket(pgp_sig_subpkt_t &subpkt)
     bool oklen = true;
     bool checked = true;
 
-    switch (subpkt.type) {
```

Comment:
Do I correctly understand, MSVC doesn't allow range case labels? Then it would be cleaner to change `PGP_SIG_SUBPKT_PRIVATE_FIRST` and `PGP_SIG_SUBPKT_PRIVATE_LAST` to `PGP_SIG_SUBPKT_PRIVATE_100, 101... 110`, and use 10 case labels. This would also be useful together with undefined behavior sanitizer.

Reply:
Exactly. Range switch turned out to be a non-standard feature.